### DISCUSSION TITLE
[A/L] (2021/22) Relazioni - Esercizio 22

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#294] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152320569-edad84db-cdda-4049-866b-d2a10434549d.png)

**Comment by LuginiAndrea** [comment]
Non è transitiva: $1 R^{\cup} 3, 3 R^{\cup} 4, \nexists 1 R^{\cup} 4$
$R^{R \circ S} = \left(1,4\right), \left(2,5\right)$, $R^{S \circ R} = \left(1,4\right), \left(2,5\right)$
$R^* = R \cup \left(1,3\right), \left(1, 4\right), \left(1, 5\right), \left(2,4\right), \left(2,5\right), \left(3,5\right)$

**Comment by LuginiAndrea** [comment]
Non è transitiva: $1 R^{\cup} 3, 3 R^{\cup} 4, \nexists 1 R^{\cup} 4$
$R^{R \circ S} = \left(1,4\right), \left(2,5\right)$, $R^{S \circ R} = \left(1,4\right), \left(2,5\right)$
$R^* = R \cup \left(1,3\right), \left(1, 4\right), \left(1, 5\right), \left(2,4\right), \left(2,5\right), \left(3,5\right)$
